File: 프로그래머스/0/120834. 외계행성의 나이/외계행성의 나이.java
import java.util.*;
class Solution {
    public String solution(int age) {
        String answer = "";
        int a = 0;
        
        StringBuilder sb = new StringBuilder();
        
        while(age != 0){
            a = (age % 10) + 97;

            char c = (char) a;
            sb.append(c);
            age /= 10;
        }
        
        answer = sb.reverse().toString();
        
        return answer;
    }
}

File: 프로그래머스/0/120834. 외계행성의 나이/외계행성의 나이.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

char* solution(int age) {
    int len = 0;
    int temp_age = age;
    
    while(temp_age > 0){
        len++;
        temp_age /= 10;
    }
    
    char* answer = (char*)malloc((len + 1) * sizeof(char));
    char* temp = (char*)malloc((len + 1) * sizeof(char));
    
    for(int i = 0; i < len; i++){
        temp[i] = (age % 10) + 'a';
        age /= 10;
    }
    
    for(int j = 0; j < len; j++){
        answer[j] = temp[len - j - 1];
    }
    
    answer[len] = '\0';
    
    return answer;
}